src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -19,21 +19,28 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
 
+<docs>
+This component displays the text inside the message component and can be used for
+the main body of the message as well as a quote.
+</docs>
+
 <template>
 	<div
-		class="wrapper"
+		class="message"
 		:class="{ 'hover': hover }"
 		@mouseover="hover=true"
 		@mouseleave="hover=false">
-		<div class="message">
-			<div class="message-avatar">
-				<Avatar :user="actorDisplayName" :display-name="actorDisplayName" />
+		<div v-if="isFirstMessage" class="message__author">
+			<h6>{{ actorDisplayName }}</h6>
+		</div>
+		<div class="message__main">
+			<div class="message__main__text">
+				<p>{{ message }}</p>
 			</div>
-			<slot />
-			<div v-show="isTemporary" class="message-right icon-loading-small" />
-			<div v-show="!isTemporary" class="message-right">
-				<h6>{{ messageTime }}</h6>
-				<Actions v-show="hover" class="actions">
+			<div class="message__main__right">
+				<div v-if="isTemporary" class="icon-loading-small" />
+				<h6 v-else>{{ messageTime }}</h6>
+				<Actions v-show="hover" class="message__main__right__actions">
```

Comment:
Remember that actions needs to be shown all the time if we're in mobile view :)
There is the isMobile mixin in @nextcloud/vue :)

Reply:
> Remember that actions needs to be shown all the time if we're in mobile

That would clutter up the screen quite a bit.. I think I remember @jancborchardt saying something about displaying it when you actually tap the message, I think it's a better solution :)